lua/ulx/modules/sh/cfc_propify.lua

```diff
@@ -38,7 +38,7 @@ local relativeDirFuncs = cmd.relativeDirFuncs
 local function propifyPlayer( caller, ply, modelPath, overrideHopPress, overrideHopCooldown )
     local canPropify = hook.Run( "CFC_ULX_PropifyPlayer", caller, ply, false ) ~= false
     if not IsValid( ply ) then return "Invalid player!" end
-    if not canPropify then return ply:GetNick() .. " cannot be propified!" end
+    if not canPropify then return ply:Nick() .. " cannot be propified!" end
```

Comment:
nitpick but imo it's better to use GetName as it's more logical and matches up with stuff like GetPos and whatnot

Reply:
Yeah, I prefer GetName, though most of the repo and (I think) our other addons use Nick